### DISCUSSION TITLE
HGLRC f722 Zeus VTX swith not working

### DISCUSSION THREAD

**Discussion by luqasz** [iNavFlight/inav#6872] [Q&A]
Hi

As in title, I've setup [USER1](https://github.com/iNavFlight/inav/blob/master/src/main/target/HGLRCF722/target.h#L160) mode in receiver and by flipping a switch VTX doesn't power OFF or ON. I have connected my VTX as in [diagram](https://cdn.shopifycdn.net/s/files/1/2672/8290/files/Zeus-F722_V2.1_20200720_1024x1024.jpg). INAV 2.6.1

1. Does any one experience same issue ?
2. Maybe I am doing something wrong ?

**Comment by rickiswolf** [comment]
I have the same issue. 

Maybe there is an issue with the mapping for the pins. (wrong pin is assigned to the switch?)
